packages/runtime/container-runtime/src/opProperties.ts

```diff
@@ -12,7 +12,7 @@ export const opSize = (op: ISequencedDocumentMessage): number => {
         op.contents :
         JSON.stringify(op.contents);
     const data = opHasData(op) ? op.data : "";
-    return content.length + data.length;
+    return content?.length + data.length;
```

Comment:
What does undefined + a number give? JS is notorious for doing weird stuff in cases like that I would be more explicit about that case

Reply:
NaN